src/Microsoft.AspNetCore.Sockets.Client.Http/LongPollingTransport.cs

```diff
@@ -121,18 +123,24 @@ public Task StartAsync(Uri url, IDuplexPipe application, TransferFormat transfer
                         Log.ReceivedMessages(_logger);
 
                         var stream = new PipeWriterStream(_application.Output);
-                        await response.Content.CopyToAsync(stream);
+                        try
+                        {
+                            await _response.Content.CopyToAsync(stream);
+                        }
+                        catch { /*First request*/};
```

Comment:
Why would this throw? If it's an expected condition, can we check it before calling `CopyToAsync`?

Reply:
This call to copytoasync was throwing on the first request. We should be keeping track of the first request on the client side transport as well and add that check here.